bundles/com.espressif.idf.core/META-INF/MANIFEST.MF

```diff
@@ -30,5 +30,8 @@ Export-Package: com.espressif.idf.core,
  com.espressif.idf.core.util,
  org.json.simple,
  org.json.simple.parser
-Bundle-ClassPath: lib/json-simple-1.1.1.jar,
- .
+Bundle-ClassPath: .,
+ lib/json-simple-1.1.1.jar,
+ lib/commons-collections4-4.4.jar,
+ lib/commons-io-2.9.0.jar,
+ lib/commons-lang3-3.12.0.jar
```

Comment:
I could see this is added in the Ui plugin as well, we can keep it here and remove it in other places. We can export packages required from the core plugin

Reply:
done